pkg/quota/image/imagestreamimport_evaluator.go

```diff
@@ -127,29 +132,35 @@ func (c *imageStreamImportUsageComputer) Usage(object runtime.Object) kapi.Resou
 
 func (c *imageStreamImportUsageComputer) getImagesUsageIncrement(isi *imageapi.ImageStreamImport) kapi.ResourceList {
 	specRefsIncrement := resource.NewQuantity(0, resource.DecimalSI)
-	allRefsIncrement := resource.NewQuantity(0, resource.DecimalSI)
+	statusRefsIncrement := resource.NewQuantity(0, resource.DecimalSI)
 
 	for _, imageSpec := range isi.Spec.Images {
```

Comment:
Looking at the code here, it's extremely unlikely that any of these are going to point to images.  Users will overwhelmingly be importing images by tag - so the majority of the time this is going to calculate to statusRefsIncrement == 0, which is wrong.  Instead, we need the increment to be `number_of_refs_in_isi - number_of_id_refs_from_isi_that_already_exist_in_image_streams`.  However, given that the second number is going to be close to zero most of the time, why bother?  If we just returned `number_of_refs_in_isi`, what's the impact to users?  The problem here is acting on the ISI doesn't have the info you need - only acting on the resulting object does.

If we only looked at number of refs total, then the only calculation that matters would be the current value of the total increment (and you could, I guess, continue to refresh the full list of image streams in order to be more accurate).